server/routes/routes.js

```diff
@@ -9,15 +9,12 @@ const {
 
 const router = express.Router();
 
-router.use('/login', express.static('app/login'));
+router.get('/login', express.static('app/login'));
```

Comment:
Is this change a leftover from experimentation, or intentional?

(I have no qualms about it, in fact I prefer `get` over `use` if it makes sense to; just gotta point out when accidental changes sneak in)

Reply:
I believe this is an error. I found it later on and I've switched it back.

Reply:
Should be changed back in the next PR.